packages/framework-provider-aws/src/library/events-adapter.ts

```diff
@@ -88,7 +88,37 @@ export async function storeEvents(
   config: BoosterConfig,
   logger: Logger
 ): Promise<void> {
-  // TODO: Manage query pagination and db.batchWrite limit of 25 operations at a time
+  const batchSize = 25
```

Comment:
Is there any constant or function in the SDK that brings us this value? Having this hardcoded here feels a bit fragile...